packages/angular/cli/models/architect-command.ts

```diff
@@ -88,8 +88,30 @@ export abstract class ArchitectCommand extends Command<ArchitectCommandOptions>
       const projectNames = this.getProjectNamesByTarget(this.target);
       const { overrides } = this._makeTargetSpecifier(options);
       if (projectNames.length > 1 && Object.keys(overrides || {}).length > 0) {
-        throw new Error('Architect commands with multiple targets cannot specify overrides.'
-          + `'${this.target}' would be run on the following projects: ${projectNames.join()}`);
+        // Verify that all builders are the same, otherwise error out (since the meaning of an
+        // option could vary from builder to builder).
+
+        let isSameBuilder = true;
+        let builder: string | null = null;
+        for (const projectName of projectNames) {
+          const targetSpec: TargetSpecifier = this._makeTargetSpecifier(options);
+          const targetDesc = this._architect.getBuilderConfiguration({
+            project: projectName,
+            target: targetSpec.target,
+          });
+
+          if (builder === null) {
+            builder = targetDesc.builder;
+          }
+          if (targetDesc.builder !== builder) {
+            isSameBuilder = false;
+          }
+        }
+
+        if (!isSameBuilder) {
+          throw new Error('Architect commands with multiple targets cannot specify overrides.'
```

Comment:
Can you change this error message to indicate that the problem is not the multiple targets, but rather that they have different builders?

Reply:
Done.